### DISCUSSION TITLE
Podman "Error: no registries found in registries.conf, a registry must be provided" while logging/pulling from docker.io

### DISCUSSION THREAD

**Discussion by YevheniiPokhvalii** [containers/podman#16103] [General]
How to integrate `podman` with `docker.io` by default?

According to these articles: 
https://www.mankier.com/5/containers-registries.conf
https://www.redhat.com/sysadmin/manage-container-registries
you can configure a default registry by adding the file `$HOME/.config/containers/registries.conf`.

I added this file and filled it out with these details:
```
$ cat .config/containers/registries.conf
[[registry]]
location = "docker.io"
```

When I try to log in to DockerHub with `podman`, I am getting:
```
$ podman login
Error: no registries found in registries.conf, a registry must be provided
```
So I have to login explicitly with:
```
$ podman login docker.io
```
of with:
```
podman login -u user -p pass -v docker.io
```

The same situation is with pulling a custom docker image:
```
$ podman pull epamedp/codebase-operator
Error: short-name "epamedp/codebase-operator" did not resolve to an alias and no unqualified-search registries are defined in "/home/user/.config/containers/registries.conf"
```

And again I have to indicate the registry explicitly:
```
$ podman pull docker.io/epamedp/codebase-operator
```

Could you help with this configuration?
P.S. Also, according to the article, it is advisable to indicate `docker.io/library` as a registry `prefix` and I also did not figure out how to do it.
I tried to modify `/etc/containers/registries.conf` as well. Still, the same issue.
I am using:
```
$ podman --version
podman version 3.4.4
```

**Comment by YevheniiPokhvalii** [comment]
I am answering my question.
I managed to fix the issue by modifying `$HOME/.config/containers/registries.conf` with one line:
```
$ cat .config/containers/registries.conf
unqualified-search-registries = ["docker.io"]
```

**Comment by baude** [comment]
may we close this?

**Comment by YevheniiPokhvalii** [comment]
> may we close this?

Sure, you can close it or convert it to discussion in case there is some advice on this matter.
